webpack/assets/javascripts/react_app/components/HostsIndex/BulkActions/reassignHostGroup/BulkReassignHostgroupModal.js

```diff
@@ -0,0 +1,194 @@
+  fetchHostgroups,
+  HOSTGROUP_KEY,
+} from './actions';
+import { visit } from '../../../../../foreman_navigation';
+import { foremanUrl } from '../../../../common/helpers';
+
+const BulkReassignHostgroupModal = ({
+  isOpen,
+  closeModal,
+  selectedCount,
+  orgId,
+  fetchBulkParams,
+}) => {
+  const dispatch = useDispatch();
+  const [hostgroupId, setHostgroupId] = useState('');
+  const hostgroups = useSelector(state =>
+    selectAPIResponse(state, HOSTGROUP_KEY)
+  );
+  const hostgroupStatus = useSelector(state =>
+    selectAPIStatus(state, HOSTGROUP_KEY)
+  );
+  const hostUpdateStatus = useSelector(state =>
+    selectAPIStatus(state, BULK_REASSIGN_HOSTGROUP_KEY)
+  );
+  const handleModalClose = () => {
+    setHostgroupId('');
+    closeModal();
+  };
+
+  useEffect(() => {
+    dispatch(fetchHostgroups());
+  }, [dispatch]);
+
+  const handleError = ({ response }) => {
+    handleModalClose();
+    dispatch(
+      addToast(
+        failedHostsToastParams({
+          ...response.data.error,
+          key: BULK_REASSIGN_HOSTGROUP_KEY,
+        })
+      )
+    );
+  };
+  const handleSave = () => {
+    const requestBody = {
+      included: {
+        search: fetchBulkParams(),
+      },
+      hostgroup_id: hostgroupId,
+    };
+
+    dispatch(
+      bulkReassignHostgroups(
+        requestBody,
+        () => visit(foremanUrl('/new/hosts')),
+        handleError
+      )
+    );
+  };
+
+  const modalActions = [
+    <Button
+      key="add"
+      ouiaId="bulk-reassign-hg-modal-add-button"
+      variant="primary"
+      onClick={handleSave}
+      isDisabled={hostUpdateStatus === STATUS.PENDING}
+      isLoading={hostUpdateStatus === STATUS.PENDING}
+    >
+      {__('Save')}
+    </Button>,
+    <Button
+      key="cancel"
+      ouiaId="bulk-reassign-hg-modal-cancel-button"
+      variant="link"
+      onClick={handleModalClose}
+    >
+      {__('Cancel')}
+    </Button>,
+  ];
+  return (
+    <Modal
+      isOpen={isOpen}
+      onClose={handleModalClose}
+      onEscapePress={handleModalClose}
+      title={__('Change host group')}
+      width="50%"
+      position="top"
+      actions={modalActions}
+      id="bulk-reassign-hg-modal"
+      key="bulk-reassign-hg-modal"
+      ouiaId="bulk-reassign-hg-modal"
+    >
+      <TextContent>
+        <Text ouiaId="bulk-reassign-hg-options">
+          <FormattedMessage
+            defaultMessage={__(
+              'Change the host group of {hosts}. Some hosts may already be in your chosen host group.'
+            )}
+            values={{
+              hosts: (
+                <strong>
+                  <FormattedMessage
+                    defaultMessage="{count, plural, one {# {singular}} other {# {plural}}}"
+                    values={{
+                      count: selectedCount,
+                      singular: __('selected host'),
+                      plural: __('selected hosts'),
+                    }}
+                    id="ccs-options-i18n"
```

Comment:
`id` should be unique on the page, per HTML specs. So it would cause problems if both of those rendered to the screen at the same time.

Typically `id` should be an (abbreviated) description of what the element is. I'm guessing `ccs` is a copy-paste that traveled from "change content source."

Reply:
Ack..That seems to be the case..Will update the Id in both places..